### PR TITLE
Aarch64 support

### PR DESCRIPTION
**Pull request by tdewey-rpi**
Just enough to use AppImageCraft on a Raspberry Pi 5.

### CODE REVIEW

**Comment by tdewey-rpi** [comment]
Tagging @TheAssassin for visibility. Looks like your project (like mine) isn't auto-assigning.

**Comment by TheAssassin** [comment]
I'd consider auto-assigning someone on a personal project superfluous. I'm watching this repository anyway.

I'd prefer having everything in a single workflow file. It makes sense to run all the tests on all platforms. I'll have a look at this ASAP. It looks to be doable rather painlessly.

It further looks like Miniconda is finally(?) available for 64-bit ARM, which I think was not the case a few years ago.

**Comment by tdewey-rpi** [comment]
I can confirm that with the patch to linuxdeploy-plugin-conda, this PR lets me build the Appimagecraft AppImage as-is.

I suspect there's a patch to unifying the build scripts by allowing an $ARCH parameter - they appear to be broadly consistent, and so long as Shellcheck never changes how it refers to aarch64 (for example, to favour arm64) the change looks fairly straightforward.

File: appimagecraft-aarch64.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+appimage:
+  arch: aarch64
+
+  linuxdeploy:
+    plugins:
+      - conda
+    extra_args: -i "$PROJECT_ROOT"/deployment/appimagecraft.svg -d "$PROJECT_ROOT"/deployment/appimagecraft.desktop --custom-apprun "$PROJECT_ROOT"/deployment/AppRun.sh
+    environment:
+      PIP_REQUIREMENTS: .
+      ARCH: aarch64
```
Comment (tdewey-rpi) [inline appimagecraft-aarch64.yml:26]: This would be obviated by #13, I believe.

Comment (TheAssassin) [reply appimagecraft-aarch64.yml:26]: Shouldn't be needed anyway, as the value is exported from the `arch` key in #18. Have you tried without it?

**Review by TheAssassin — APPROVED** [review]

**Comment by TheAssassin** [comment]
What's missing from this PR is a CI job to provide the binaries. I'll merge this for now, please don't hesitate to create another PR to add said job. Thanks!

**Pull request merged**